Applied-Research-Winter-2022/Pozter
# Poz 79 navbar update and buttons routing

## Link Jira ticket:
https://jess-nguy.atlassian.net/browse/POZ-79

## Author Notes:
- added router links to buttons in Canvas, SocialMedia and UserInput vues.
- connected the toolbar with the side navbar.
- unified the button colors in all vues.
- replaced the logo with a 🅿️ link. 
- renamed the routes to show the on the toolbar.

## Preview:
![1Home](https://user-images.githubusercontent.com/62397382/159139147-4ca8fbc8-fa61-4897-9063-2e192873ba78.jpg)



![2Billboard](https://user-images.githubusercontent.com/62397382/159139154-ceb3eca6-50e8-4884-9f98-15eb1e82ef0e.jpg)

![3Canvas](https://user-images.githubusercontent.com/62397382/159139161-e3a53096-7d92-4cf1-9681-13d54870ec4c.jpg)

![4UserContent](https://user-images.githubusercontent.com/62397382/159139166-38e12784-56a0-4ec6-8702-db361f29cfdd.jpg)

![7Navbar](https://user-images.githubusercontent.com/62397382/159139359-f2ab75eb-97df-49b8-9341-6f50364d3e4a.jpg)

![5SocialMadia](https://user-images.githubusercontent.com/62397382/159139170-26fbae01-a796-46f3-8faa-bd610d76f0af.jpg)

![6Success](https://user-images.githubusercontent.com/62397382/159139176-861be916-3f14-452a-b254-fbf82a712f18.jpg)

## Impacts and Notes:
- the sidebar links are not clickable yet.

## Testing Guidance:
- open "localhost:8080" to see home and go from there.
- check if the navbar is visible across all views.
- check if the "Next", "Cancel", "Upload Screen Canvas", "Delete Screen Canvas", "Create Another Screen", and "Configure Social Media" buttons work.

## Author Checklist:
- [x] Change Jira ticket status to into Review.
- [n/a] Updated the documentation.
